src/pages/donate.astro

```diff
@@ -0,0 +1,165 @@
+---
+
+import Layout from '../layouts/Layout.astro';
+import Container from "../components/layout/Container.astro";
+import ExternalLogo from "../components/layout/ExternalLogo.astro";
+
+---
+
+<Layout title="Donate">
+  <div class="bg-nixdarkblue px-12 pt-16 pb-16 md:pb-0 text-white">
+    <Container class="grid px-12">
+      <h1 class="text-5xl font-heading font-bold leading-none text-center md:text-left">
+        Donate
+      </h1>
+    </Container>
+  </div>
+  <img src="/divider/mountain_nixdarkblue.svg" class="w-full hidden md:block" />
```

Comment:
- this can be turned into a separate component with variable header text, name suggestion maye `PageHeader` or `PageHero`
- the blue background should start roughly under the NixOS logo and stretch to the right of the screen